FAIL: test_umath.TestComplexFunctions.test_branch_cuts

Got following error when running "runtests.py" on linux 64 bit.
```
[mlai@begws92 numpy]$ uname -a
Linux begws92 2.6.32-358.23.2.el6.x86_64 #1 SMP Sat Sep 14 05:32:37 EDT 2013 x86_64 x86_64 x86_64 GNU/Linux


======================================================================
FAIL: test_umath.TestComplexFunctions.test_branch_cuts(<ufunc 'arccosh'>, [-1, 0.5], [1j, 1j], 1, -1, True)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/mlai/anaconda/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/home/mlai/github/numpy/build/testenv/lib/python2.7/site-packages/numpy/core/tests/test_umath.py", line 1690, in _check_branch_cut
    assert_(np.all(np.absolute(y0.imag - yp.imag) < atol), (y0, yp))
  File "/home/mlai/github/numpy/build/testenv/lib/python2.7/site-packages/numpy/testing/utils.py", line 53, in assert_
    raise AssertionError(smsg)
AssertionError: (array([  0.00000000e+00+3.14159265j,   1.11022302e-16-1.04719755j]), array([  4.71216091e-07+3.14159218j,   1.28119737e-13+1.04719755j]))

----------------------------------------------------------------------
Ran 5974 tests in 45.985s

FAILED (KNOWNFAIL=3, SKIP=3, failures=1)
```

--- a/numpy/core/src/private/npy_config.h
+++ b/numpy/core/src/private/npy_config.h
@@ -93,6 +93,12 @@
 #undef HAVE_CATANH
 #undef HAVE_CATANHF
 #undef HAVE_CATANHL
+#undef HAVE_CACOS
+#undef HAVE_CACOSF
+#undef HAVE_CACOSL
+#undef HAVE_CACOSH
+#undef HAVE_CACOSHF
+#undef HAVE_CACOSHL
 #endif
 #undef TRIG_OK
 

